android_core_components/src/org/ros/android/NodeMainExecutorService.java

```diff
@@ -215,16 +214,6 @@ public int onStartCommand(Intent intent, int flags, int startId) {
     if (intent.getAction() == null) {
       return START_NOT_STICKY;
     }
-    if (intent.getAction().equals(ACTION_START)) {
-      Preconditions.checkArgument(intent.hasExtra(EXTRA_NOTIFICATION_TICKER));
-      Preconditions.checkArgument(intent.hasExtra(EXTRA_NOTIFICATION_TITLE));
-      Intent notificationIntent = new Intent(this, NodeMainExecutorService.class);
-      notificationIntent.setAction(NodeMainExecutorService.ACTION_SHUTDOWN);
-      PendingIntent pendingIntent = PendingIntent.getService(this, 0, notificationIntent, 0);
-      Notification notification = buildNotification(intent, pendingIntent);
-
-      startForeground(ONGOING_NOTIFICATION, notification);
-    }
```

Comment:
Just to clarify: I don't mind the notification from appearing when a ROS application is launched, and it's fine if it can be shut down by tapping it. What I don't find consistent with today's apps is the fact that when you close the app by swiping it it doesn't really close _unless_ you tap the notification.

In other words, I'd keep the notification, but close the app when swiping it or when tapping the notification.
Is that possible, or do we need to remove the notification completely?

Reply:
Yes it's possible. I'll make this change.